packaging/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -87,6 +87,10 @@ spec:
               value: quay.io/strimzi/kaniko-executor:latest
             - name: STRIMZI_DEFAULT_MAVEN_BUILDER
               value: quay.io/strimzi/maven-builder:latest
+            - name: STRIMZI_OPERATOR_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
```

Comment:
Is this needed if you set it in the startup script?

Reply:
Good point, I've documented it instead.